src/app/components/admin/page-admin-years/page-admin-years.component.html

```diff
@@ -0,0 +1,54 @@
+<ksi-admin-sidebar></ksi-admin-sidebar>
+<div class="content-container">
+    <div class="content">
+
+        <h2>{{ 'admin.years.title' | translate }}</h2>
+        <a class="btn btn-ksi"
+            [routerLink]="['/', routes.routes.admin._, routes.routes.admin.years._, routes.routes.admin.years.edit, '0']">
+            {{ icon.ADD }} {{ 'admin.years.newYear' | translate }}</a>
+
+        <table class="table table-striped table-hover table-responsive-md threads table-dark">
```

Comment:
Místo table-dark používej prosím `ksiTheme`, které automaticky nastaví světlý/tmavý mód

```suggestion
        <table class="table table-striped table-hover table-responsive-md threads" ksiTheme>
```